### PR TITLE
Monitoring formulas

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/11066

### CODE REVIEW

File: modules/salt/pages/formula-monitoring.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+.Procedure: Configuring the Grafana Formula
+. Navigate to the menu:Formulas[Grafana] tab, and set these parameters in the [guimenu]``Grafana`` section:
+* Check the [guimenu]``Enabled`` box to enable Grafana visualizations.
+* In the [guimenu]``Default admin user`` field, type the name of the default Grafana user.
+* In the [guimenu]``Default admin password`` field, enter a password for the default user.
+Alternatively, click btn:[Generate new password] to generate a password and fill the field.
+. For each Prometheus data source you want to use, in the menu:Datasources[Prometheus] section, click btn:[+], and set these parameters:
+* In the [guimenu]``Datasource name`` field, type a name to identify the data source.
+* In the [guimenu]``Prometheus URL`` field, type the location of the Prometheus server, and append port 9090.
```
Comment (keichwa) [inline modules/salt/pages/formula-monitoring.adoc]:
```suggestion
* In the [guimenu]``Prometheus URL`` field, type the location of the Prometheus server, and append port ``9090``.
```
Looks better this way, IMO.  It's also a "literal" string.

File: modules/salt/pages/formula-monitoring.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+. Click btn:[Save Formula] to save your configuration.
+
+
+
+
+.Procedure: Configuring the Prometheus Exporters Formula
+. Navigate to the menu:Formulas[Prometheus Exporters] tab, and set these parameters in the [guimenu]``Node Exporter`` section:
+* Check the [guimenu]``Enabled`` box to enable the node exporter.
+* In the [guimenu]``Arguments`` field, type any customized arguments for this exporter.
+For example, ``--web.listen-address=":9100"``.
```
Comment (keichwa) [inline modules/salt/pages/formula-monitoring.adoc]: is the colon actually required here?  if yes, I'd rate it as an implementation issue.

File: modules/salt/pages/formula-monitoring.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+. In the [guimenu]``Apache Exporter`` section:
+* Check the [guimenu]``Enabled`` box to enable the Apache exporter.
+* In the [guimenu]``Arguments`` field, type any customized arguments for this exporter.
+For example, ``--telemetry.address=":9117"``.
+. In the [guimenu]``Postgres Exporter`` section:
+* Check the [guimenu]``Enabled`` box to enable the Postres exporter.
```
Comment (keichwa) [inline modules/salt/pages/formula-monitoring.adoc]:
```suggestion
* Check the [guimenu]``Enabled`` box to enable the PostreSQL exporter.
```
And raise a bug asking dev to fix the string in the UI.

File: modules/salt/pages/formula-monitoring.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+* In the [guimenu]``Data source Name`` field, type the name of the data source to use.
```
Comment (keichwa) [inline modules/salt/pages/formula-monitoring.adoc]: add to the bug: wrong usage of title style.

File: modules/salt/pages/formula-monitoring.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/salt/pages/formula-monitoring.adoc:70]: Dunno, I copied it from the UI.

**Pull request merged**